src/plugins/remove/RemoveAction.js

```diff
@@ -34,8 +34,8 @@ export default class RemoveAction {
     invoke(objectPath) {
         let object = objectPath[0];
         let parent = objectPath[1];
-        this.showConfirmDialog(object).then(() => {
-            this.removeFromComposition(parent, object);
+        this.showConfirmDialog(object).then(async () => {
+            await this.removeFromComposition(parent, object);
```

Comment:
Let's talk about whatever test failure necessitated this change.

Reply:
I think this will be fixed from the above suggestion to use the composition api correctly.